content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -44,6 +44,8 @@ Example: `99% of the time requests should have a latency of less than 300ms over
 
 While the SLO remains above the target percentage, the SLO's status will be displayed in green font. When the target percentage is violated, the SLO's status will be displayed in red font. You can also optionally include a warning percentage that is greater than the target percentage to indicate when you are approaching an SLO breach. When the warning percentage is violated (but the target percentage is not violated), the SLO status will be displayed in yellow font.
 
+**Note:** The number of decimal places you can specify for your monitor-based SLOs differs depending time windows you choose. Up to 2 decimal places are allowed for 7-day and 30-day targets, and up to 3 decimal places are allowed for 90-day targets. The precision shown in the details UI of the SLO will be up to `num_target_decimal_places`, meaning 2 decimal places for 7-day and 30-day targets and 3 decimal places for 90-day targets. The reason these limits are in place is because your monitor-based SLO error budget is specified in terms of time. A 99.999% target for a 7-day and 30-day targets would result in error budgets of 6 seconds and 26 seconds, respectively. Monitors evaluate every minute, thus the granularity of a monitor-based SLO is also 1 minute. This means one alert would fully consume and overspend the 6 second or 26 second error budgets in the previous examples (for use cases requiring more granularity, consider using [metric-based SLOs][7] instead). Furthermore, error budgets this small are likely unrealistic goals for you to try to achieve in practice.
```

Comment:
```suggestion
**Note:** The number of decimal places you can specify for your monitor-based SLOs differs depending on the time windows you choose. Up to two decimal places are allowed for 7-day and 30-day targets, and up to three decimal places are allowed for 90-day targets. The precision shown in the details UI of the SLO will be up to `num_target_decimal_places`, meaning two decimal places for 7-day and 30-day targets and two decimal places for 90-day targets. These limits are in place because monitor-based SLO error budget is specified in terms of time. A 99.999% target for a 7-day and 30-day targets results in error budgets of 6 seconds and 26 seconds, respectively. Monitors evaluate every minute, thus the granularity of a monitor-based SLO is also 1 minute. This means one alert would fully consume and overspend the 6 second or 26 second error budgets in the previous examples (for use cases requiring more granularity, consider using [metric-based SLOs][7] instead). Furthermore, error budgets this small are likely unrealistic goals for you to try to achieve in practice.
```